avm/res/cache/redis/main.bicep

```diff
@@ -94,7 +94,7 @@ param tenantSettings object = {}
 param zoneRedundant bool = true
 
 @description('Optional. If the zoneRedundant parameter is true, replicas will be provisioned in the availability zones specified here. Otherwise, the service will choose where replicas are deployed.')
-param zones array = []
+param zones array = [1,2,3]
```

Comment:
```suggestion
param zones int[] = [1,2,3]
```

Reply:
Also please note a merge conflict sneaked its way in

Reply:
Fixed.